### PR TITLE
e2e tests for the opening balance

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR add : 
- e2e tests for the opening balance setting 
- fix `period_id` of some transaction `data.sql`

### CODE REVIEW

**Comment by mbayopanda** [comment]
@jniles could you review this PR

File: client/src/partials/fiscal/fiscal.openingBalance.html [hunk 1]
```diff
@@ -39,13 +39,13 @@
           <div ng-class="{'text-success': FiscalOBCtrl.balanced, 'text-danger': !FiscalOBCtrl.balanced}">
             <span ng-show="OpeningAccountForm.$submitted" >
               <i class="fa" ng-class="{'fa-check': FiscalOBCtrl.balanced, 'fa-warning': !FiscalOBCtrl.balanced}"></i>
-              <span ng-show="FiscalOBCtrl.balanced">{{ 'ACCOUNT.BALANCED' | translate }}</span>
-              <span ng-show="!FiscalOBCtrl.balanced && FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_BALANCED' | translate }}</span>
-              <span ng-show="!FiscalOBCtrl.balanced && !FiscalOBCtrl.hasPositive">{{ 'ACCOUNT.NOT_POSITIVE' | translate }}</span>
+              <span data-status="balanced" ng-show="FiscalOBCtrl.balanced">{{ 'ACCOUNT.BALANCED' | translate }}</span>
```
Comment (jniles) [inline client/src/partials/fiscal/fiscal.openingBalance.html:42]: :+1:  Cool!  Very readable.

File: server/controllers/finance/fiscal.js [hunk 2]
```diff
@@ -321,7 +321,7 @@ function setOpeningBalance(req, res, next) {
   .then(hasPrevious => {
 
     if (hasPrevious) {
-      // load from the period N+1 into period 0
+      // load from the period N+1 of the year N-1 into period 0 of the year N
```
Comment (jniles) [inline server/controllers/finance/fiscal.js:324]: Nice clarification :+1:

File: test/end-to-end/fiscalYears/fiscalYears.spec.js [hunk 3]
```diff
@@ -75,4 +75,88 @@ describe('Fiscal Year', function () {
     components.modalAction.confirm();
     components.notification.hasSuccess();
   });
+
+  it('set the opening balance for the first fiscal year', () => {
+    helpers.navigate(path);
```
Comment (jniles) [inline test/end-to-end/fiscalYears/fiscalYears.spec.js:80]: I think it automatically navigates in a `before()` block.  I could be wrong though.

**Review by jniles — APPROVED** [review]
Very clear tests.  This LGTM.

**Pull request merged**